source/clear-linux/get-started/virtual-machine-install/vmware-player.rst

```diff
@@ -1,116 +1,528 @@
+#########################################################
 
-This section explains how to run Clear Linux OS for Intel® Architecture
-within a `VMware Player`_ environment.
+`VMware Workstation 14 Player`_ is a type 2 hypervisor which runs on top of another
+operating system such as Windows or Linux.  Creation, configuration, management, and execution of virtual
+machines are done on the local system.  For the cloud version of VMware, see 
+:ref:`vmware-esxi`.
 
-Please ensure you have enabled `Intel® Virtualization Technology
-<http://www.intel.com/content/www/us/en/virtualization/virtualization-technology/intel-virtualization-technology.html>`_
-(Intel® VT) and `Intel® Virtualization Technology for Directed I/O
-<https://software.intel.com/en-us/articles/intel-virtualization-technology-for-directed-io-vt-d-enhancing-intel-platforms-for-efficient-virtualization-of-io-devices>`_
-(Intel® VT-d) in your BIOS/UEFI firmware configuration.
+This section explains how to run |CLOSIA| as a virtual machine using the 
+VMware Workstation 14 Player hypervisor.
 
-Install VMware Player
-=====================
+There are 2 ways to create a |CL| :abbr:`VM (Virtual Machine)` to run in VMware:
 
-VMware Workstation Player, formerly VMware Player, is a virtualization
-software package for x64 computers running Microsoft Windows or Linux.
-Download VMware player from the `VMware website`_.
+* `Method #1`: Install |CL| into a new VM.  This provides flexibility 
+  in configuring the VM size, partitions, initial |CL| bundles selection, etc.
+* `Method #2`: Use a ready-made VMware |CL| image.  
 
-Player on Linux
----------------
+Both are discussed below.  
 
-For the Linux option, you should have a :file:`VMware-Player-{version}_FILE.bundle`
-file. To install it, run:
+.. note::
 
-::
+  The figures shown throughout this document are from the Windows\* version of 
+  VMware Workstation 14 Player.  The menus and prompts in these figures are 
+  similar as those for the Linux version with some minor wording differences.
 
-  $ sudo bash ./VMware-Player-{version}.x86_64.bundle
+Install the VMware Workstation Player hypervisor
+================================================
 
-Player on Windows
------------------
+#. Enable the `Intel® Virtualization Technology`_ (Intel® VT) and the
+   `Intel® Virtualization Technology for Directed I/O`_ (Intel® VT-d) in the
+   host machine’s BIOS.
+#.  `VMware Workstation 14 Player`_ is available for Windows and Linux.  Download 
+    the preferred version of choice.
+#.  Install it.
 
-Follow the instructions from the Setup Assistant.
+    * For Linux distros supported by VMware: 
 
+      #.  Enable a GUI desktop.  
+      #.  Start a terminal emulator.
+      #.  Start the installer and follow the guided steps.
 
-Prepare Image
-=============
+        .. code-block:: console
 
-#. Download the `latest`_ |CL| **live** version (clear-XXXX-live.img.xz)
+          $ sudo sh ./VMware-Player-<version>.x86_64.bundle
```

Comment:
If this is the command to start the installer, which I know it is, there is no indication of that on the steps above.